src/synthcity/plugins/core/models/tabular_ddpm/__init__.py

```diff
@@ -1,3 +1,6 @@
+# mypy: allow-untyped-defs, allow-untyped-calls
```

Comment:
You shouldn't disable the existing checks. All the other modules respect these checks. Please fix the issues instead.

Reply:
I have removed all these comments except flake8 F405, which prohibits star import. I have a utils.py and another internal script imports a lot of variables and functions from it. I wonder if it is also mandatory that I need to replace the star import with a long list of names.

Reply:
Hello @TZTsai 

Thank you so much for working on fixing the errors!
It would be recommended to import by name instead of star import. Otherwise, you could import by mistake something that hides a name in the current module, without knowing that.

Thanks again for the contribution!

Reply:
Hello @bcebere 

You are welcome! I have removed star import now as well.